test/middleware.spec.ts

```diff
@@ -43,9 +43,11 @@ describe("middleware", () => {
     return http()
       .post(`/webhook`, {
         events: [webhook],
+        destination: "Uhogehoge",
```

Comment:
I think that destination should be `U[0-9a-f]{32}` eaven test data. eg. `Uaaaabbbbccccddddeeeeffff`.

Reply:
Sounds legit.